spec/unit/provider/package/aix_spec.rb

```diff
@@ -98,6 +97,25 @@
       @provider.stubs(:latest_info).returns( { :version => "1.2.3.5" } )
       expect(@provider.latest).to eq("1.2.3.5")
     end
+
+    it "should prefetch the right values" do
+      Process.stubs(:euid).returns( 0 )
+      resource = Puppet::Type.type(:package).
+          new(:name => 'sudo.rte', :ensure => :latest,
+              :source => 'mysource', :provider => :aix)
+
+      resource.stubs(:should).with(:ensure).returns(:latest)
+      resource.should(:ensure)
+
+      resource.provider.class.stubs(:execute).returns(<<-END.chomp)
+sudo:sudo.rte:1.7.10.4::I:C:::::N:Configurable super-user privileges runtime::::0::
+sudo:sudo.rte:1.8.6.4::I:T:::::N:Configurable super-user privileges runtime::::0::
+END
+
+      resource.provider.class.prefetch('sudo.rte' => resource)
+      info = resource.provider.latest_info
+      expect(info[:version]).to eq('1.8.6.4')
```

Comment:
I think it might be clearer (and also fail more clearly without your fix) if you use just `latest`.

i.e., 
```
resource.provider.class.prefetch('sudo.rte' => resource)
expect(resource.provider.latest).to eq('1.8.6.4')
```